src/app.js

```diff
@@ -1,21 +1,23 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
 
-module.exports = sum;
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  const normolizeURL = new URL(req.url, `http://${req.headers.host}`);
+  const parts = normolizeURL.pathname.split('/');
+  const query = Object.fromEntries(normolizeURL.searchParams.entries());
+
+  res.setHeader('Content-Type', 'application/json');
+
+  res.end(JSON.stringify({
+    parts,
+    query,
+  }));
+
+  server.listen(PORT, () => {
```

Comment:
Move the server.listen call outside of the request handler. It should be on the same level as the server creation.